test/access-token.test.js

```diff
@@ -280,7 +280,7 @@ describe('loopback.token(options)', function() {
       ' when enableDoubkecheck is true',
     function(done) {
       var token = this.token;
-
+      var app = loopback({localRegistry: true, loadBuiltinModels: true});
```

Comment:
`var app = loopback();` was not working but it is working now for some reason, maybe because of the globak AccessToken thing.